doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -133,14 +143,16 @@ downloaded, a single line is needed to load it.
 
 .. topic:: **Dataset formatting: data shape**
 
-    We can find two main representations for MRI scans:
+    It is important to appreciate two main representations for
+    storing and accessing more than one nifti images, that is sets
+    of MRI scans:
 
-    - a big 4D matrix representing 3D MRI along time, stored in a big 4D
-      NifTi file.
+    - a big 4D matrix representing (3D MRI + 1D for time), stored in a single
+      nifti file.
```

Comment:
Nifti vs nifti doesn't seem to be consistent. I don't really know which one is preferable.